### ISSUE TITLE
xccdf_org.ssgproject.content_rule_ensure_gpgcheck_repo_metadata should not be enabled by default and should be 'low' severity

### ISSUE THREAD

**Issue by trevor-vaughan** [ComplianceAsCode/content#2455]
#### Description of problem:

**TL;DR;** The potential risk caused by enabling `repo_gpgcheck` outweighs any potential benefit if TLS is enabled.

In my opinion, the following check should **only** be enabled if all of your repositories are internally managed `xccdf_org.ssgproject.content_rule_ensure_gpgcheck_repo_metadata`.

The reason for this is that YUM presently does not (to my knowledge) have any way to differentiate between package signing GPG keys and repo signing GPG keys.

This means that if, for instance, I host my packages via some shared Nexus, then I have to add the Nexus GPG key to my trust list for the repo.

I fundamentally do **not** want to do this! I shouldn't be allowing my Nexus maintainer to potentially install software on my system without my explicit knowledge.

You should use TLS, and the repo should have a trusted certificate there and that should be sufficient for the metadata until RPM can tell the difference between these two certificates.

Additionally, this is marked as `high` severity and that seems to be massive overkill considering that 1) the packages are still signed and validated and 2) TLS is required.

#### SCAP Security Guide Version:

0.1.36

#### Operating System Version:

EL 7

**Comment by redhatrises** [comment]
Please open a ticket with DISA on this issue.

**Comment by trevor-vaughan** [comment]
@redhatrises Will do. Do you mind if I keep these rolling in here for tracking purposes? Perhaps add a tag of 'DISA' when it has been filed?

**Comment by redhatrises** [comment]
> @redhatrises Will do. Do you mind if I keep these rolling in here for tracking purposes? Perhaps add a tag of 'DISA' when it has been filed?

@trevor-vaughan go ahead. If you want, add the DISA ticket number in here when you get chance.

**Comment by redhatrises** [comment]
Comment from Steve Grubb on SSG mailing list

```
I'm not sure that I agree with this assessment. I am going to look into this
in more detail over the next couple of days because...the current rev of OSPP
calls out for auditing of software update integrity checks. It calls out for
integrity checks and for them to be enabled. It calls out for the vendor to
supply SCAP content for the evaluated configuration. So that means we
shouldn't be turning it off. So, if there is anything amiss, I'd be inclined
to file bz to get it fixed. Using TLS defends against a different attack
vector than the one repo_gpgcheck and gpgcheck were designed for.

So...I'll get back to everyone with what I find out.

-Steve
```

**Comment by redhatrises** [comment]
See https://lists.fedorahosted.org/archives/list/scap-security-guide@lists.fedorahosted.org/thread/ZDKOEZN3BRXRED6K3ACYEJUXRPDTPJWW/ for more discussion on this.

**Comment by kenyon** [comment]
Seems like this is a duplicate of  #1596 and  #2189.

**Comment by kenyon** [comment]
Maybe close this since the offending STIG rule was removed (#3388)?

**Comment by redhatrises** [comment]
Closing ticket since rule has been removed in latest STIG.
